yarn.lock

```diff
@@ -8925,6 +8932,13 @@ pn@^1.1.0:
   resolved "https://registry.yarnpkg.com/pn/-/pn-1.1.0.tgz#e2f4cef0e219f463c179ab37463e4e1ecdccbafb"
   integrity sha512-2qHaIQr2VLRFoxe2nASzsV6ef4yOOH+Fi9FBOVH6cqeSgUnoyySPZkxzLuzd+RYOQTRpROA0ztTMqxROKSb/nA==
 
+polished@^3.4.2:
```

Comment:
@gestchild Where are we using polished?

Reply:
https://github.com/wellcometrust/wellcomecollection.org/blob/e06b5075023758b3b5bc7c0b74e5c67d1ec806ae/common/views/components/IIIFViewer/IIIFViewer.js#L4

Reply:
Sorry - didn't see it in the `package.json`.

Do we have rules as to how this interacts with our palette as it feels like we're extending it, and assuming we don't want to proliferate how we lighten / darken, especially to make sure we meet WCAG?